packages/git/src/browser/git-scm-provider.ts

```diff
@@ -241,21 +241,33 @@ export class GitScmProvider implements ScmProvider {
                 return DiffUris.encode(
                     fromFileUri.withScheme(GIT_RESOURCE_SCHEME).withQuery('HEAD'),
                     changeUri.withScheme(GIT_RESOURCE_SCHEME),
-                    this.labelProvider.getName(changeUri) + ' (Index)');
+                    nls.localize(
+                        'theia/core/git/tabTitleIndex',
```

Comment:
Suggestion: Only files that belong to the `@theia/core` package should use `theia/core`. See the other usages of `theia/core/git` in this PR as well.
```suggestion
                        'theia/git/tabTitleIndex',
```